doc/repoman.md

```diff
@@ -0,0 +1,21 @@
+# repoman
+
+## Name
+
+repoman - Run pacman, only considering the official repositories
+
+## Synopsis
+
+repoman ARG
+
+## Description
+
+Repoman is a wrapper for pacman which removes all unofficial repositories from the configuration. All arguments given are directly passed on to pacman
```

Comment:
Well, this is true in practice. But the inverse, i.e. creating a configuration file
with only the official repositories, reflects the repoman logic more
accurately.